doc/zh/namespace.md

```
@@ -0,0 +1,79 @@
+# 名字空间
+
+## 使用场景
+- 租户隔离：系统支持百万级或千万级数量的名字空间。名字空间限制数据访问范围，实现租户间的数据隔离。
+
+## 使用流程
+
+### 1. 创建名字空间
+执行命令：
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)